pkg/mark/link.go

```diff
@@ -0,0 +1,69 @@
+package mark
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/kovetskiy/mark/pkg/confluence"
+)
+
+type Link struct {
+	MDLink string
+	Link   string
+}
+
+// ResolveRelativeLinks finds links in the markdown, and replaces one pointing to other Markdowns either by own link (if not created to Confluence yet) or witn actual Confluence link
```

Comment:
Could you wrap the comment to 80 characters per line? This long is too long, even GitHub can't fit it in the screen size.

Reply:
Wrapped those long comments.